docs/domain/COMMENT_SYSTEM.md

```
@@ -0,0 +1,55 @@
+## Thread 구조
+
+- PK = `(deck_id, thread_date)`
+- `thread_date` = 작성자의 client local date at write time
+- 작성자 nick 표시는 `{nick}#{anon_id 앞 4hex}` (IDENTITY_MODEL 참고)
+- 답글 / 멘션 / 이미지: 미지원
+
+## 가시성 게이트
+
+가림의 유일한 사유 = 데일리 라운드 결과 스포일러 방지.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)